pkgs/test_api/mono_pkg.yaml

```diff
@@ -0,0 +1,10 @@
+dart:
+ - dev
+
+stages:
+    - analyze_and_format:
+      - group:
+        - command: tool/dartfmt_and_analyze.sh
```

Comment:
this doesn't need to be a group if its just one command - but note that if you do make it a group with the two explicit commands it might actually be a bit nicer:

- it will show in travis the exact dartfmt/dartanalyzer commands being ran
- it will tell you which one failed, and always run both instead of exiting when the first one fails

Reply:
Done.